Demo/Demo/Models/SetupTokenResponse.swift

```diff
@@ -8,9 +8,22 @@ struct SetUpTokenResponse: Decodable, Equatable {
     
     let id, status: String
     let customer: Customer?
+    let links: [Link]
+    var paypalURL: String? {
+        if let link = links.first(where: { $0.rel == "approve" }) {
+            let url = link.href
+            return url
+        }
+        return nil
+    }
```

Comment:
```suggestion
    var paypalURL: String? {
        return links.first { $0.rel == "approve" }?.href
    }
```
🧶 : ^ would be technically equivalent.

Reply:
😅

```suggestion
   var paypalURL: String? {
        links.first { $0.rel == "approve" }?.href
    }
```

Reply:
Oh nice haha. 🙌